### ISSUE TITLE
Error: rpc error: code = Unavailable desc = transport is closing

### ISSUE THREAD

**Issue by jvandenbos** [datarootsio/terraform-module-azure-datalake#171]
Strange - keep getting this error:

```
null_resource.sql_init[0]: Still creating... [10s elapsed]
null_resource.sql_init[0] (local-exec): -1
azurerm_cosmosdb_sql_container.metadata: Still creating... [40s elapsed]
null_resource.sql_init[0]: Still creating... [20s elapsed]
null_resource.sql_init[0] (local-exec): -1
null_resource.sql_init[0]: Creation complete after 23s [id=7882090812782899888]
azurerm_cosmosdb_sql_container.metadata: Still creating... [50s elapsed]
azurerm_cosmosdb_sql_container.metadata: Still creating... [1m0s elapsed]
azurerm_cosmosdb_sql_container.metadata: Creation complete after 1m3s [id=/subscriptions/b970f71b-bc04-47c8-89cd-7e491b4656b2/resourceGroups/rgcdedlplay02/providers/Microsoft.DocumentDB/databaseAccounts/cmdbcdedlplay02/sqlDatabases/metadatadb/containers/metadata]

Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing



Error: rpc error: code = Unavailable desc = transport is closing
```

**Comment by sdebruyn** [comment]
Probably related to https://github.com/databrickslabs/terraform-provider-databricks/issues/223

**Comment by sdebruyn** [comment]
Will be fixed with #176 (expected in the next few days)
